src/IonCatalog.ts

```diff
@@ -21,33 +21,45 @@ import { SharedSymbolTable } from "./IonSharedSymbolTable";
     }
 
     /**
-    * A catalog holds available shared symbol tables and always includes the system symbol table.
-    * @see http://amzn.github.io/ion-docs/symbols.html#the-catalog
-    */
+     * A catalog holds available shared symbol tables and always includes the system symbol table.
+     * @see http://amzn.github.io/ion-docs/symbols.html#the-catalog
```

Comment:
This URL redirects (and drops the anchor reference).  Please use http://amzn.github.io/ion-docs/docs/symbols.html#the-catalog instead.

Reply:
Umm, just to make sure I read this right, should it be http***s***?

Reply:
either http or https works, the difference is whether the path is `ion-docs` vs `ion-docs/docs`

Reply:
Ah, got it, thanks--I couldn't see it, but I fixed this by copy/pasting the URL over what was there.